rcjaRegistration/events/views.py

```diff
@@ -234,3 +236,103 @@ def delete(self, request, teamID=None, attendeeID=None, eventID=None):
         # Delete team
         eventAttendance.delete()
         return HttpResponse(status=204)
+
+def getEventsForSummary(state, year):
+    """ Create list of event dictionaries of all events in state and year """
+    eventList = Event.objects.filter(state = state, year = year)
+
+    # Find information for events
+    events = []
+    for event in eventList:
+        eventDict = {}
+        eventDict["name"] = event.name
+
+        if event.startDate==event.endDate:
+            eventDict["date"] = event.startDate
+        else:
+            eventDict["date"] = f"{event.startDate.strftime('%d/%m/%Y')} - {event.endDate.strftime('%d/%m/%Y')}"
```

Comment:
It looks like multi day events render the date a bit differently, should be an easy fix I think - Either of the date formats are fine.
![image](https://github.com/user-attachments/assets/8b3d0107-966b-4646-87ec-c9d51912df58)

Also while you are here, it would be good to reverse the sort of the list such that it is in the order that the events were run (oldest at the top)